starters/vue3-apollo-quasar/src/variables/counts.js

```diff
@@ -0,0 +1,3 @@
+import { makeVar } from '@apollo/client';
+
+export const counts = makeVar(0);
```

Comment:
If the way forward is to keep this as an abstraction, there are 2 main suggestions:
- Rename the `variables` directory to something like `globals` or `store` to indicate that it is a directory that holds global state.

- Move mutation functionality into this directory. No point in decoupling it from this and coupling it to the `NumberCounter` component. _View more comments on the same in the `NumberCounter` component review._

Reply:
changing variables to store doesn't look good as it defeats what the Reactive variable is.

Reply:
`store` was an example. What I'm trying to illustrate is that we need a name that is more self-descriptive. 
What does `variables` mean? Is this a convention? Since this will be used by many people, we do not want to introduce new conventions that are not necessarily known by the general community.

The suggestion of `store` or `globals` carries forward user pre-existing knowledge

Reply:
Renamed to globals

Reply:
variables is kinda a convention, but globals is fine